### PR TITLE
update-autofill-documentation CSH-7040: update autofill documentation…

### PR DESCRIPTION
**Pull request by wwsno**
… to better reflect what is supported. Note from 11.77 (TBD) ExtraMetadata will be supported.

### CODE REVIEW

File: docs/COMPONENTS.md [hunk 1]
```diff
@@ -61,16 +61,15 @@ Components use the following definition:
             // (Introduced in version 1.1.0)
             // (Optional) Define if the content of a component should be filled automatically.
             // For example: automatically fill the caption of an image when an image is added to an article.
-            // Currently only works with Enterprise metadata only.
+            // Currently only works with Studio server metadata only.
```
Comment (wwjhu) [inline docs/COMPONENTS.md:64]:
```suggestion
            // Currently only works with Studio server metadata.
```

File: docs/COMPONENTS.md [hunk 2]
```diff
@@ -61,16 +61,15 @@ Components use the following definition:
...
+            // Only Studio server metadata of plain string types are supported. Other types will show
+            // undefined behavior (such as multi lists).
             //
-            // Set different trigger options:
-            // (Introduced in version 1.1.0, deprecated since version 1.2.0)
-            //   Once - means that it will only be triggered the first time
-            //   (in the case of filling a caption: when the image is added for the first time)
+            // From Studio client 11.77 "ExtraMetadata" is supported as well (with the same type restrictions).
```
Comment (wwjhu) [inline docs/COMPONENTS.md:68]: An ExtraMetadata example would be nice.

**Review by wwjhu — APPROVED** [review]

File: docs/COMPONENTS.md (hunk 2 above)
Comment (wwsno) [reply docs/COMPONENTS.md:68]: Done.

**Review by wwjhu — APPROVED** [review]
🧇

**Review by ruudjungbacker — APPROVED** [review]
👍

**Pull request merged**